docs/src/simulation_tutorial.md

```diff
@@ -0,0 +1,678 @@
+The `θ` vector is the flattened version of the variance-covariance matrix - a lowertrinangular matrix.
+The on-diagonal elements are just the standard deviations (the `σ`'s), If all off-diagonal elements are zero, we can use our
+calculation above. The off-diagonal elements are covariances and correspond to the correlations (the `ρ`'s).
+If they are unequal to zero, as it is in our `kb07`-dataset, we cannot recreate the variance-covariance matrix having the model output.
+We just take it from the model we already fitted
+
+See the two inner values:
+```@example Main
+kb07_m.θ
+```
+
+### Play with theta
+```@example Main
+kb07_m2 = kb07_m
+update!(kb07_m2, re_item, re_subj)
+
+
+# make a lower triangular matrix
+
+re_item = create_re(0.5363168233715857,0.37133693708531357)
+re_item[2]=-0.70
+re_item
+
+re_subj = create_re(0.4382528181348316)
+
+
+make the compact form out of it = is equal to θ
+newθ= vcat( flatlowertri(re_item), flatlowertri(re_subj) )
+```
+
+
+## *A simple example*
+
+Having this knowledge about the parameters we can now **simulate data from scratch**
+
+The `simdat_crossed()` function from `MixedModelsSim` lets you set up a data frame with a specified experimental design.
+For now, it only makes fully balanced crossed designs!, but you can generate an unbalanced design by simulating data for the largest cell and deleting extra rows.
+
+
+Firstly we will set an easy design where `subj_n` subjects per `age` group (O or Y) respond to `item_n` items in each of two `condition`s (A or B).
+
+Your factors need to be specified separately for between-subject, between-item, and within-subject/item factors using `Dict` with the name of each factor as the keys and vectors with the names of the levels as values.
+
+First we have to define factors in a dict.
+
+We start with the between subject factors:
+```@example Main
+subj_btwn = Dict("age" => ["O", "Y"])
+```
+
+There are no between-item factors in this design so you can omit it or set it to nothing.
+Note that if you have factors which are between subject and between item, you need to put them in both dicts.
+```@example Main
+item_btwn = nothing
+```
+
+Next, we put within-subject/item factors in a dict:
+```@example Main
+both_win = Dict("condition" => ["A", "B"])
+```
+
+Define subject and item number:
+ ```@example Main
+subj_n = 10
+item_n = 30
+```
+
+Simulate data:
+```@example Main
+dat = simdat_crossed(subj_n,
+                     item_n,
+                     subj_btwn = subj_btwn,
+                     item_btwn = item_btwn,
+                     both_win = both_win);
+```
+
+Have a look:
+```@example Main
+first(DataFrame(dat),8)
+```
+The values we see in the column `dv` is just random noise.
+
+Set contrasts:
+```@example Main
+contrasts = Dict(:age => HelmertCoding(),
+                 :condition => HelmertCoding());
+```
+
+Define formula:
+```@example Main
+f1 = @formula dv ~ 1 + age * condition + (1|item) + (1|subj);
```

Comment:
I stumbled here, why condition is not a random slope of the effects. O.c. you can simulate it like that, and it will be easier with the thetas - but this was confusing for me and maybe a comment would be helpful?

Reply:
Sure, I'll put a note referring to the explanation of theta